custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml

```diff
@@ -36,7 +38,14 @@ card_media_player:
   state:
     - operator: "template"
       value: "[[[ return entity.state != 'off' ]]]"
-      name: "[[[ return states[entity.entity_id].attributes.media_title; ]]]"
+      name: |
+        [[[
+          var name = states[entity.entity_id].attributes.friendly_name;
+          if(states[entity.entity_id].attributes.media_title){
+            var name = states[entity.entity_id].attributes.media_title;
```

Comment:
JS Nitpick: No need to use `var name` again. Also, it would be better to use `let name` in the first place.